IntegrationTests/UITests/UITest.cs

```diff
@@ -1,58 +1,127 @@
 ﻿using System;
 using System.Linq;
+using System.Reflection;
 using System.Threading.Tasks;
 using System.Windows.Forms;
+using CommonTestUtils;
 using GitUI;
+using NUnit.Framework;
 
 namespace GitExtensions.UITests
 {
     public static class UITest
     {
-        public static async Task WaitForIdleAsync()
+        public static void RunForm<T>(
```

Comment:
📝 The changes in this file are blocked on #7623

Reply:
I have removed this temporary -- "WIP" in PR title -- experiment.
Still "WIP" due to unclear cause for timeout of `WaitForPendingOperations` (see below).